gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-bin.zip
```

Comment:
I think we should use the "-all" distribution.

Reply:
can you explain why that's advantageous?

Reply:
It includes not only the binaries but also the sources and docs.